colossalai/zero/utils/zero_hook.py

```diff
@@ -77,11 +54,10 @@ def post_fwd_exec(self, module: torch.nn.Module, *args):
                 tensor_list.append(param.colo_attr.sharded_data_tensor)
             self.shard_strategy.shard(tensor_list, self.process_group)
 
-        # remove torch payload
-        for param in module.parameters(recurse=False):
-            param.colo_attr.set_data_none()
-
-    def pre_bwd_exec(self, module: torch.nn.Module, input, output):
+    def payload_movement(self, module: torch.nn.Module):
```

Comment:
The name is bad. Actually, you did tensor moving, tensor state changing and mem sampling.

Reply:
May be `adjust_model_data`?

Reply:
that looks better